rq/worker.py

```diff
@@ -376,6 +376,8 @@ def kill_horse(self, sig=SIGKILL):
         """
         try:
             os.kill(self.horse_pid, sig)
+            self.log.debug('Horse killed')
```

Comment:
Can you add the PID number of the killed process in the log message? Also please change the severity level to log.info.

Reply:
> 
> 
> Can you add the PID number of the killed process in the log message? Also please change the severity level to log.info.

Ok, I have modified the code.